app/Console/Commands/DemandPreparePayments.php

```diff
@@ -42,31 +43,45 @@ public function handle(): void
             return;
         }
 
-        $groupedEvents = $events->groupBy('pay_to');
+        $groupedEvents = $events->each(function (EventLog $entry) {
+            $entry->licence_fee = (int)round($entry->event_value * Config::fetch(Config::LICENCE_TX_FEE));
+            $entry->operator_fee = (int)round($entry->event_value * Config::fetch(Config::OPERATOR_TX_FEE));
+            $entry->paid_amount = $entry->event_value - $entry->licenceFee - $entry->operatorFee;
+        })->groupBy('pay_to');
```

Comment:
Are you sure that we want to half up values? As I remember Paweł uses `floor` function for this operation.
Also, amounts calculation looks different. I'd suggest to discuss it together and move to one place (source code) if it should be the same.